libpod/pod.go

```diff
@@ -693,6 +752,11 @@ func (p *Pod) Status() (map[string]ContainerStatus, error) {
 	return status, nil
 }
 
+// MakePauseContainer returns whether the pod will create a pause container
+func (p *Pod) MakePauseContainer() bool {
```

Comment:
Rename to HasPauseContainer()

Reply:
Do you think the variable to store it should be HasPauseContainer as well? i.e: p.config.PauseContainer.HasPauseContainer

Reply:
Sure